scripts/auto-versioning.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+VERSION=$(curl -s https://api.github.com/repos/snyk/driftctl/releases/latest | grep tag_name | cut -d '"' -f 4 | sed 's/v//')
+MINOR_VERSION="$(echo "$VERSION" | sed 's/\.[[:digit:]]\+$//g')"
+
+echo "Running script for version $VERSION (minor: $MINOR_VERSION)"
+
+# Check if the version already exist
+if cat versions.json | grep -q $VERSION &>/dev/null; then
+  echo "Skipping new version $VERSION since it already exists"
+  exit 0
+fi
+
+# Check if the version is a patch release
+if cat versions.json | grep $MINOR_VERSION -q &>/dev/null; then
```

Comment:
```suggestion
if cat versions.json | grep "$MINOR_VERSION" -q &>/dev/null; then
```

As per previous thread about quoting and shellcheck. There are other lines this applies to, too.